frontend/src/lib-common/api-types/income.ts

```diff
@@ -2,18 +2,9 @@
 //
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
-export const incomeCoefficients = [
-  'MONTHLY_WITH_HOLIDAY_BONUS',
-  'MONTHLY_NO_HOLIDAY_BONUS',
-  'BI_WEEKLY_WITH_HOLIDAY_BONUS',
-  'BI_WEEKLY_NO_HOLIDAY_BONUS',
-  'DAILY_ALLOWANCE_21_5',
-  'DAILY_ALLOWANCE_25',
-  'YEARLY'
-] as const
-
-export type IncomeCoefficient = (typeof incomeCoefficients)[number]
```

Comment:
These are customized by some municipalities in their forks, so they have to be kept for the time being.

Reply:
this ok? https://github.com/espoon-voltti/evaka/pull/4168/commits/5a8a66d135b55b256bf091fcd662844831cf92e4